osu.Game/Screens/Edit/Compose/Components/EditorSelectionHandler.cs

```diff
@@ -176,7 +176,8 @@ private void createStateBindables()
         /// </summary>
         protected virtual void UpdateTernaryStates()
         {
-            SelectionNewComboState.Value = GetStateFromSelection(SelectedItems.OfType<IHasComboInformation>(), h => h.NewCombo);
+            if (SelectedItems.Any())
+                SelectionNewComboState.Value = GetStateFromSelection(SelectedItems.OfType<IHasComboInformation>(), h => h.NewCombo);
```

Comment:
I know that I proposed this originally but now I see that it's not correct in all instances. This change means that if you select an object that starts a combo, the new combo toggle will enable. Worse yet, if you select a combination of new-combo and non-new-combo objects, the toggle will enter indeterminate state and not even deselection will get rid of that state.

Not sure what the solution here is. Maybe the interaction of this popover with the new combo toggle should be removed completely. @peppy maybe you will have a UX opinion on this

Reply:
@bdach correct me if i'm wrong, but it seems the only change here is that the check for selected items has been added. so the fail case would be that "new combo" may not be reset when the selection is empty?

Reply:
Correct - in the worst case it will stay in indeterminate state after selecting and deselecting a group of objects with mixed new combo state.

Reply:
I think this is okay.

Reply:
I've just made an alternative change to just revert the indeterminate state to full false on deselect (https://github.com/ppy/osu/pull/30214/commits/88aea70429513151076b97dd8a2790aaedc9d27c). Should hopefully be uncontroversial.

My only bother has ever been it visually looking like in the "indeterminate" state after a deselect, nothing more.